calico/openstack/t_etcd.py

```diff
@@ -17,10 +17,25 @@
 
 # Etcd-based transport for the Calico/OpenStack Plugin.
 
+# Standard Python library imports.
```

Comment:
Imports are supposed to be split into three groups &ndash; standard library, related third-party, and local application &ndash; with a newline between each. Since the `etcd` and `eventlet` libraries aren't part of the standard library, they should be split into their own import group. More like:

```python
# Standard library imports
import json
import re
import time

# Related third-party imports
import etcd
import eventlet

# Calico imports
```

There's more detail about how import statements should be formatted [in PEP 8](https://www.python.org/dev/peps/pep-0008/#imports).